### DISCUSSION TITLE
Report upload to DefectDojo

### DISCUSSION THREAD

**Discussion by cepxeo** [aquasecurity/trivy#1716] [Q&A]
Hi, we're using DefectDojo for issues aggregation and Trivy 'vulnerability' reports export in json format works perfectly fine. However the 'config' checks are not uploaded. Guess the fields naming is different. Are you planning to support config report upload to DefectDojo or otherwise do you support any other issue aggregators where we can upload the config reports?

**Comment by knqyf263** [comment]
If I understand correctly, DefectDojo needs to support misconfigurations. You may want to ask them.
